README.md

```diff
@@ -189,6 +189,47 @@ symbolic link should have a relative path.
       path: zshrc
 ```
 
+If the source location is omitted or NULL, dotbot will use the destination instead, without a leading `.` if present. This makes the following three config files equivalent:
```

Comment:
I think the "without a leading `.` present" by itself doesn't exactly specify what happens -- maybe mention that you take the basename without the leading `.` present?